packages/pulp/python-setuptools-scm/python-setuptools-scm.spec

```diff
@@ -0,0 +1,56 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name setuptools-scm
+
+Name:           python-%{pypi_name}
+Version:        3.3.3
+Release:        1%{?dist}
+Summary:        the blessed package to manage your versions by scm tags
+
+License:        MIT
+URL:            https://github.com/pypa/setuptools_scm/
+Source0:        https://files.pythonhosted.org/packages/source/s/%{pypi_name}/setuptools_scm-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+
+%description
+setuptools_scm setuptools_scm handles managing your Python package versions in
+SCM metadata instead of declaring them as the version argument or in a SCM
+managed file.It also handles file finders for the supported SCMs. setup.py
+usage To use setuptools_scm just modify your project's setup.py file like
+this:* Add setuptools_scm to the setup_requires parameter. * Add the
```

Comment:
Do we care about a readable description?

Reply:
IMHO not, this one is actually almost readable.